includes/admin/settings/class-wc-settings-general.php

```diff
@@ -285,7 +285,10 @@ private function store_notice_setting_moved_notice() {
 
 			<p><?php
 			/* translators: %s: URL to customizer. */
-			echo wp_kses( sprintf( __( 'Looking for the store notice setting? It can now be found <a href="%s">in the Customizer</a>.', 'woocommerce' ), esc_url( admin_url( 'customize.php?url=' . wc_get_page_permalink( 'shop' ) . '&autofocus[panel]=woocommerce' ) ) ), array(
+			echo wp_kses( sprintf( __( 'Looking for the store notice setting? It can now be found <a href="%s">in the Customizer</a>.', 'woocommerce' ), esc_url( add_query_arg( array(
+				'autofocus[panel]' => 'woocommerce',
```

Comment:
Since is using `add_query_arg()` can change it too:

```php
array(
    'autofocus' => array( 
        'panel' => 'woocommerce',
    ),
    'url'       => wc_get_page_permalink( 'shop' ),
)
```